indico/modules/events/editing/client/js/components/Timeline/ResetReview.jsx

```diff
@@ -0,0 +1,74 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import resetReviewsURL from 'indico-url:event_editing.api_undo_review';
+
+import React, {useState} from 'react';
+import {useSelector, useDispatch} from 'react-redux';
+import PropTypes from 'prop-types';
+import {Icon} from 'semantic-ui-react';
+import {Translate} from 'indico/react/i18n';
+
+import * as selectors from '../../selectors';
+import {resetReviews} from '../../actions';
+
+import RequestConfirm from '../RequestConfirm';
+import './ResetReviews.module.scss';
+
+export default function ResetReview({revisionId}) {
+  const {eventId, contributionId, editableType} = useSelector(selectors.getStaticData);
+  const [submitting, setSubmitting] = useState(false);
+  const [isOpen, setIsOpen] = useState(false);
+
+  const dispatch = useDispatch();
+
+  const resetRevisions = async () => {
+    setSubmitting(true);
+    await dispatch(
+      resetReviews(
+        resetReviewsURL({
+          confId: eventId,
+          contrib_id: contributionId,
+          type: editableType,
+          revision_id: revisionId,
+        })
+      )
+    );
+    setSubmitting(false);
+  };
+  return (
+    <>
+      <Icon
```

Comment:
Maybe a `Popup` on the `Icon` would be nice as well with a simple text like `Undo review`.